src/main/java/duke/commands/Deadline.java

```diff
@@ -0,0 +1,58 @@
+package duke.commands;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Encapsulates a Deadline object that implements a Task and has a deadline.
+ *
+ * @author Owen Tan
+ * @version Duke Level-9
+ */
+public class Deadline extends Task {
+    protected LocalDateTime by;
+
+    /**
+     * Constructor for Deadline.
```

Comment:
Would it be clearer to separate description and params by an empty line?

Reply:
This modification can be considered for several other places as well.

Reply:
Point taken! Thank you.